### ISSUE TITLE
[CONSIDER] Multiple photos

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#15718]
## Issue Description
**Current experience**
VAMC editors can add a single image to events and stories.  Images are not required.  
Images are included in the Story list. 

**Desired experience**
VAMC editors frequently request the ability to add more than one image to stories. 
> The sizes of cropping are not intuitive. 7:2 ratios are not a normal ratio and creates major issues for us to identify best imaging. In addition to the oddness of the sizing of images, we are limited to one image for a story. Having the ability to add a second or third image to a story will help us in better communicating with our Veterans.

Another use case: Facilities are allowed only one facility image but could often better support wayfinding for first time visitors if they could add more than one. (To be handled separately)

**Background**
Historically we have resisted adding more photos because 
  - we want to ensure that people have a responsive experience on low band-width devices or for those who pay for data
  - current design/style/industry comparisons: if you look at CNN.com or other major news sites, times have changed, and they generally only have one big image.
  - the fewer images, the fewer problems we have with alt-text
 
The editorial experience for embedding photos into stories is not ideal and likely needs a comprehensive review and improvement but we need to enable additional photos in the near term. 
- Background: Enhance editorial experience for embedding photos into stories department-of-veterans-affairs/va.gov-team#13906

**The Secretary has emphasized storytelling, supported by photos and videos.** 
How might we enable the use of more than one image for VAMC stories?

## Related
- [CONSIDER] Multiple photos per Vet Center location department-of-veterans-affairs/va.gov-team#33945
- Prevent editing of VAMC system banner decorative images department-of-veterans-affairs/va.gov-team#8233

## Questions

- What is the maximum recommended number of images for a single story or event?
- How does page render if only one image is included vs multiple?

---
## Tasks
- [ ] Review current Story and image embed editorial experience 
- [ ] Review selection of VAMC story pages on front end to understand common uses and implications of cropping
- [ ] Update Story design to allow more than one image 
- [ ] Determine optimum photo size/resolution for design

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by davidmpickett** [comment]
@mmiddaugh transferred this to the CMS repo and added to UX agenda - FYI in case you were accessing with direct url

**Comment by davidmpickett** [comment]
@mmiddaugh @xiongjaneg @laflannery 
Given that the CMS Platform team is currently working on enhancements to the alt text experience, maybe it makes sense to set #13103 as a blocker to this? That would be one way to better ensure accessibility even with multiple photos.

**Comment by laflannery** [comment]
I wouldn't set that as a blocker, I don't think it would be a good use of time to necessarily wait on all of that. What I think might be more efficient would be to instead for you all to move forward on discovery, investigation, etc. on your end and then if we find a particular alt text piece that the CMS team is already working on we need for this work we can break it out and make it a blocker and see if they can move that up in priority. I just think that the entire epic might be too long of a wait and I think moving forward on something, knowing that there are stakeholders pushing for this, might be better. Thoughts?

**Comment by mmiddaugh** [comment]
Quote from [Reddit thread](https://www.reddit.com/r/VeteransBenefits/comments/17ovewh/whats_the_hardest_part_about_applying_for_benefits/) via Martha:

> I’m very rural by choice. However getting almost any online interactions with the VA almost requires a trip into town to the library. We only have our phones as there is no internet available yet. There’s satellite but it’s expensive for what it ends up being.
> I wish they had just a bare bones mobile app without waiting for piles of graphics, headers, backgrounds and the same spiel on every page.

**Comment by aklausmeier** [comment]
@mmiddaugh - Danielle and I discussed and echo Martha's concern re: Veterans located in rural areas having internet bandwidth issues especially if they rely on poor to moderate cell service strength. 

Possible solutions to discuss/explore:

- allow one main image and one small supporting image on Events and Stories only
- if possible, run as special pilot experiment, monitor issues/feedback
- turn off capability if problems are discovered
- provide guidance to editors on images (size, [images should follow VA guidance](https://dvagov.sharepoint.com/sites/oititsccreativephotography/), not stock imagery)
- investigate if Drupal can assist with file size compression for accessibility
- alt text is required

**Comment by xiongjaneg** [comment]
Related feedback on help text https://dsva.slack.com/archives/C0FQSS30V/p1701204924522119

**Comment by jilladams** [comment]
@davidmpickett FYI that there are 3 design tickets in this epic. One was already in Next Refinement; I've moved the other 2 over, since Michelle has noted this is a higher UX priority in Q2 than Vet Center 2.0 designs. We should maybe touch on this if there's time to look at refining any in UX Sync, after we confirm the Q2 priorities?

**Comment by davidmpickett** [comment]
> @davidmpickett FYI that there are 3 design tickets in this epic. One was already in Next Refinement; I've moved the other 2 over, since Michelle has noted this is a higher UX priority in Q2 than Vet Center 2.0 designs. We should maybe touch on this if there's time to look at refining any in UX Sync, after we confirm the Q2 priorities?

Definitely won't have time to get to these in UX Sync today given all the higher priority items. I added them to the Facilities Team Refinement Ticket #16686

**Comment by jilladams** [comment]
We reversed priority during UX sync, documented here: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17669#issuecomment-2035662432. This is back to lower priority than Vet Centers 2.0

**Comment by Agile6MSkinner** [comment]
@mmiddaugh should we try to get this on the roadmap for Q3?

**Comment by mmiddaugh** [comment]
For future reference: [Drupal CMS Image Upload Research Findings](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/medical-centers/research/2024%20VAMC%20CMS/ImageUploader/PostStudy/Findings)
